config/seata-config-etcd3/src/main/java/io/seata/config/etcd3/EtcdConfiguration.java

```diff
@@ -240,7 +238,7 @@ private static Client getClient() {
     /**
      * the type config change notifier
      */
-    private class EtcdListener implements ConfigurationChangeListener {
+    private static class EtcdListener implements ConfigurationChangeListener {
```

Comment:
Why use static?

Reply:
EtcdListener/NacosListener/ZKListener does not have access to the enclosing instance. It uses less space.